.github/workflows/unitctl.yml

```diff
@@ -15,6 +15,50 @@ permissions:
   contents: write
 
 jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        working-directory: tools/unitctl
+    env:
+      MAKE: make
+      CARGO: cargo
+      VERSION:
+      SHORT_VERSION:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - build: linux-x86_64
+            os: ubuntu-latest
+            target: x86_64-unknown-linux-gnu
+          - build: macos-x86_64
+            os: macos-latest
+            target: x86_64-apple-darwin
```

Comment:
`macOS-latest` is actually `aarch64` now by default. Would it make sense to just build that? Might be faster?

By default a macOS arm machine can build and execute either arm64 or amd64 thanks to [Rosetta](https://en.wikipedia.org/wiki/Rosetta_(software)#Rosetta_2).

Reply:
Fair enough! I think it absolutely would be reasonable!